static/app/views/dashboards/datasetConfig/metrics.tsx

```diff
@@ -382,7 +383,7 @@ function getMetricRequest(
       },
     ] as any);
   }
-  const per_page = limit && Number(limit) >= 10 ? limit : 10;
+  const actualLimit = limit && Number(limit) >= 10 ? limit : 10;
```

Comment:
I'm lacking context here, but curious - why can't we pass a lower limit than 10?

Reply:
also lacking context, maybe @obostjancic knows more?

Reply:
I am not 100% sure but if i remember correctly setting a limit that is lower than the number of groups returned would make the api responses inconsistent, so different groups would get returned on each call. It would also happen that the API errors out in case that the limit is too high instead of reducing the granularity. The actual number 10 is probably an arbitrary threshold we set. 

However this param is completely ignored by the new metrics layer so we can soon delete this altogether. Maybe lets add a comment there.